scripts/prepare_sector_network.py

```diff
@@ -4959,16 +4959,18 @@ def add_industry(
             parse_dates=True,
         )
 
-        # Add temporal loads to the network for each node
-        for node in industrial_elec_profiles.columns:
-            # The profile is already in MW for each hour
-            n.add(
-                "Load",
-                f"{node} industry electricity",
-                bus=node,
-                carrier="industry electricity",
-                p_set=industrial_elec_profiles[node],  # MW per hour
-            )
+        industrial_loads = industrial_elec_profiles.rename(
+            columns=lambda x: f"{x} industry electricity"
+        )
+
+        n.add(
+            "Load",
+            industrial_loads.columns,
+            bus=industrial_elec_profiles.columns.tolist(), 
```

Comment:
This is not very safe because for it to function properly, `industrial_loads` and `industrial_elec_profiles` must be in order. Just align the indices, then you do not need to work with lists.